src/server/connection.rs

```diff
@@ -1761,6 +1821,22 @@ impl Connection {
                 }
             }
         } else if self.authorized {
+            #[cfg(target_os = "windows")]
+            if !self.checked_multiple_session {
+                self.checked_multiple_session = true;
+                if crate::platform::is_installed()
+                    && crate::platform::is_share_rdp()
+                    && !*CONN_COUNT.lock().unwrap() > 1
```

Comment:
when the second client connects, the session chosen dialog still pops up, it's not a big problem, but if the user choose a session different from the current one, the two clients will reconnect in a loop.

Reply:
Why does CONN_COUNT not work, it has been used in other places too. Is there anything else I can use to ensure only one connection is allowed for multiple session dialog?

> And could you update the workflow on the top of the pr? Describe critical steps in more detail, including conditions, what messages to send, disconnection and reconnection.

How about this image?
<img width="1674" alt="Screenshot 2024-02-07 at 11 29 19 AM" src="https://github.com/rustdesk/rustdesk/assets/73148455/74a28cd9-1341-4d1d-805f-4862c139e661">

Reply:
`CONN_COUNT` is 2 in log.

![1707293779678](https://github.com/rustdesk/rustdesk/assets/14891774/148e004d-d885-48c7-ac0b-6689c64d5ac8)
![1707293816911](https://github.com/rustdesk/rustdesk/assets/14891774/e2f9ac50-1dfe-4eaf-a45b-6c61ee6a5d9d)
![1707293875005](https://github.com/rustdesk/rustdesk/assets/14891774/728d460a-e852-4965-b119-c37087fccfa8)